### DISCUSSION TITLE
[Task]: Where is train.py located

### DISCUSSION THREAD

**Discussion by fffcompu** [open-edge-platform/anomalib#1884] [Q&A] [answered]
### What is the motivation for this task?

Where is train.py located

### Describe the solution you'd like

Where is train.py located

### Additional context

Where is train.py located

**Comment by blaz-r** [comment, marked as answer]
Hi, in v1 train.py is no longer used. You can train via CLI or via API. Refer to [readme](https://github.com/openvinotoolkit/anomalib?tab=readme-ov-file#-training) for quick guide, or to [docs](https://anomalib.readthedocs.io/en/latest/index.html) for more info.

**Comment by fffcompu** [comment]
Thank you very much for your answer
